charts/plgd-hub/values.yaml

```diff
@@ -2527,7 +2527,7 @@ snippetservice:
           keyFile:
           certFile:
           useSystemCAPool: false
-    resourceUpdater:
+    resourceAggregate:
       cleanUpExpiredUpdates: "0 * * * *"
```

Comment:
This is a resourceUpdater setting though. I guess we should have both? ResourceAggregate and ResourceUpdater configs?

Reply:
this is a client to `resourceAggregate` -> what you want to move to `ResourceUpdater` ? just `cleanUpExpiredUpdates: "0 * * * *"`

Reply:
`CleanUpExpiredUpdates` and `ExtendCronParserBySeconds` should be outside of `resourceAggregate`. I guess we can move them to the root level if we want to avoid making a new resourceUpdaterConfig struct?

Reply:
We have something similar here:
![image](https://github.com/user-attachments/assets/d8b3fc0a-7e50-41cd-b099-01cd9930237a)

https://github.com/plgd-dev/hub/blob/5f8588a972a8b95709f8af8266e4095b6fc82db7/charts/plgd-hub/values.yaml#L2247

Maybe we can leave it as it is. What do you thing?

Reply:
moved to storage config